README.Rmd

```diff
@@ -20,7 +20,8 @@ knitr::opts_chunk$set(
 [![codecov](https://codecov.io/gh/ekamau/bayesianTiterCalc/branch/master/graph/badge.svg?token=61D5K98II1)](https://codecov.io/gh/ekamau/bayesianTiterCalc)
 <!-- badges: end -->
 
-bayesianTiterCalc is a Bayesian inference method that calculates a serum sample's antibody concentration and titer based on data from a standard neutralization assay. The method uses a logistic function in a statistical model to simulate mortality in cell culture as a function of antibody concentration. Given the limited information per sample that's inherent in the experimental design of neutralization assays (number of replicates per dilution and number of dilutions), the advantage of Bayesian inference here is the use of probability distributions to incorporate uncertainty in the outcome.
+bayesianTiterCalc is a Bayesian inference method that calculates a serum sample's antibody concentration, $\phi$, and titer based on data from a standard neutralization assay. The method uses a logistic function in a statistical model to simulate mortality in cell culture as a function of antibody concentration. <br>
+Given the limited information per sample that's inherent in the experimental design of neutralization assays (number of replicates per dilution and number of dilutions), the advantage of Bayesian inference here is the use of probability distributions to incorporate uncertainty in the outcome.
```

Comment:
I would change, "the advantage of Bayesian inference here is the use of probability distributions to incorporate uncertainty in the outcome." to "the advantage of Bayesian inference here is that it produces estimates incorporating uncertainty."